controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -300,7 +303,6 @@ function execute($request) {
 					$publishedArticle = $publishedArticleDao->newDataObject();
 					$publishedArticle->setId($submission->getId());
 					$publishedArticle->setIssueId($issueId);
-					$publishedArticle->setDatePublished(Core::getCurrentDate());
```

Comment:
You should differentiate/consider the right case (if the submission has just been re/assigned to an issue) further down, in the line 348, when you set the published date for the submission ($submission->setDatePublished($this->getData('datePublished'));). In that case, instead of assigning it the value of the input text field, use the current date (Core::getCurrentDate()). Does that work?